civihr_employee_portal/src/Forms/OnboardingWizardCustomizationForm.php

```diff
@@ -190,4 +210,17 @@ private function adjustSaveButton($form) {
 
     return $form;
   }
+
+  /**
+   * Returns the structure for the button which saves and go to test
+   *
+   * @return array
+   */
+  private function saveAndTestOutThisForm() {
```

Comment:
I'm not sure this is a good name for this function, it doesn't "save and test" this form, it returns the button for this, so maybe something like 'getSaveAndTestButton'

Reply:
Totally agree.